atom_tools/lib/converter.py

```diff
@@ -368,8 +367,6 @@ def create_paths_item(self, filename: str, paths_dict: Dict) -> Dict:
         paths_object: Dict = {}
 
         for ep in set(endpoints):
-            if ep.startswith('/ftp(?!/quarantine)'):
```

Comment:
@prabhu Yep, I actually left this in there when I fixed negative lookaheads - I have removed from the filtering branch.